packages/react-inbox/src/components/Messages2.0/actions/index.tsx

```diff
@@ -111,28 +118,41 @@ const MessageActions: React.FunctionComponent<{
     >
       <div
         className={classNames({
-          visible: isMessageHovered,
+          hidden: !isMessageActive,
         })}
       >
-        {!read && (
-          <MarkRead label={labels?.markAsRead} onClick={handleEvent("read")} />
-        )}
-        {read && (
-          <MarkUnread
-            label={labels?.markAsUnread}
-            onClick={handleEvent("unread")}
+        <TimeAgo
+          tabIndex={0}
```

Comment:
In general, setting the tabindex of an element to 0 may cause confusion for users who rely on keyboard navigation.

Reply:
the point of adding tabIndex is to allow people with keyboard access to access the created time, before they could  not